<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MAViewer2</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Desktop/Simji.png</normaloff>../Desktop/Simji.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="matplotlibWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>801</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fbButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>158</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>FaceBook</string>
    </property>
   </widget>
   <widget class="QPushButton" name="appleButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>158</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Apple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>158</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Lush (박나은 전용)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>109</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>109</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>MA20</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>109</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>MA60</string>
    </property>
   </widget>
   <widget class="QPushButton" name="teslaButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>158</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tesla</string>
    </property>
   </widget>
   <widget class="QPushButton" name="samsungButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>158</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Samsung</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>109</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>MA5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>580</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addEventButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Event</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Me"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionAbout_Me">
   <property name="text">
    <string>About Me</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>matplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidgetFile</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
